src/status_im/contexts/communities/actions/accounts_selection/style.cljs

```diff
@@ -5,7 +5,8 @@
 (def screen-horizontal-padding 20)
 
 (def container
-  {:flex 1})
+  {:flex       1
+   :margin-top -12})
```

Comment:
Any reason why we set `-12`?

Reply:
Sure, Will do!

Reply:
Hey Mohamed, Do you think it's better for the bottom-sheet changes to be a followup?
For the sake of easing QA review and minimizing this PR's scope.